Classes/Emogrifier.php

```diff
@@ -298,7 +306,17 @@ public function emogrify()
 
         $this->copyCssWithMediaToStyleNode($cssParts, $xmlDocument);
 
-        return $xmlDocument->saveHTML();
+        if ($this->returnBodyContentOnly) {
```

Comment:
The only way to do that without incurring excessive performance cost would be to refactor the vast majority of the `emogrify()` method into a new (internal) method --taking and returning a DOMDocument-- and have both the "output" functions `emogrify()` and `emogrifyBody()` (or whatever it'll be called) call that new method.

This approach would allow more extendability too, as it separates the processing from the interface.

This would be fine by me, but would it suit you?